users/ignoring.rst

```diff
@@ -140,3 +142,42 @@ all files and directories called "foo", ending in a "2" or starting with
   ``some/directory/`` matches the content of the directory, but not the
   directory itself. If you want the pattern to match the directory and its
   content, make sure it does not have a ``/`` at the end of the pattern.
+
+Ignoring files across all devices
+---------------------------------
+
+Although ``.stignore`` file is not synced between the devices, as mentioned
+previously, it's possible to take advantage of an ``#include`` pattern
+to create a single file that will hold shared ignoring patterns.
+
+Assuming the content of a file ``.stignore-shared`` is::
+
+    // Shared ignoring patterns
+    (?d).DS_Store
+    foo
+
+This file can be synced across all devices and helps with ignoring
+``.DS_Store`` if you sync a folder between devices where at least one is
+Mac OS and will also cause ``foo`` pattern being ignored on all devices.
+
+The important part is to ``#include`` this shared ignore file per each
+device so that::
+
+    // .stignore for 1st Device
+    #include .stignore-shared
+
+    // .stignore for 2nd Device
+    #include .stignore-shared
+
+    ...
+
+    // .stignore for N-th Device
+    #include .stignore-shared
+
+The shared ``#include`` does not exclude the possibility to add custom
+patterns for single devices.
```

Comment:
The `#include` directive does not prevent using additional ignore patterns applicable only to the local device.
Additional ignore patterns can be added before or after the include directive adjusting the order in which the patterns are processed.